### PR TITLE
[14.0][ADD]project_duplicated_task

### PR DESCRIPTION
**Pull request by Kev-Roche**
Allow to close a task and categorize it as a duplicate of another one.

### CODE REVIEW

File: project_duplicated_task/models/project_task.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        comodel_name="project.task",
+        relation="duplicated_closed_rel",
+        column1="duplicated_closed_id",
+        column2="task_duplicated_id",
+        string="Duplicate Closed Task",
+        readonly=True,
+        store=True,
+        ondelete="restrict",
+    )
+
```
Comment (sebastienbeau) [inline project_duplicated_task/models/project_task.py]:
```suggestion
    task_duplicated_open_ids = fields.Many2many(
        comodel_name="project.task",
        relation="project_task_duplicate_rel",
        column1="main_id",
        column2="duplicated_id",
        string="Duplicate Open Task",
        store=True,
        ondelete="restrict",
    )

    task_duplicated_closed_ids = fields.Many2many(
        comodel_name="project.task",
        relation="project_task_duplicate_rel",
        column1="duplicated_id",
        column2="main_id",
        string="Duplicate Closed Task",
        readonly=True,
        store=True,
        ondelete="restrict",
    )

```

File: project_duplicated_task/models/project_task.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    task_duplicated_id = fields.Many2one(comodel_name="project.task", string="Task")
```
Comment (sebastienbeau) [inline project_duplicated_task/models/project_task.py]: useless

File: project_duplicated_task/models/project_task.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+    is_duplicate = fields.Boolean(default=False)
+    have_duplicate = fields.Boolean(default=False)
```
Comment (sebastienbeau) [inline project_duplicated_task/models/project_task.py]:
```suggestion
    is_duplicate = fields.Boolean(default=False)
    have_duplicate = fields.Boolean(default=False)
```
should be computed field if really needed

**Comment by leemannd** [comment]
Work seems legit. @sebastienbeau Do you want to update the review and we get this merged?

**Comment by leemannd** [comment]
@Kev-Roche Do you have time to rebase your branch? the goal would be to fetch the latest changes and allow runboat to be triggered.

**Comment by Kev-Roche** [comment]
@leemannd thank you, I rebased it.

**Review by leemannd — APPROVED** [review]
Thank you.
Waiting a second review and/or an update of the existing one

**Comment by Kev-Roche** [comment]
cc @sebastienbeau

**Pull request closed without merging**